### DISCUSSION TITLE
Clipboard removed

### DISCUSSION THREAD

**Discussion by Ansarmughal99** [florisboard/florisboard#2640] [Q&A] [answered]
After the latest update pin clipboard and Saved clipboard removed.

**Comment by tsiflimagas** [comment, marked as answer]
There's a note regarding this issue in the v0.4.2 release and it's fixed in 0.4.3. Sadly, if you've already updated to v0.4.2 and don't have a backup of your clipboard content, there's no way to recover it.

**Reply by Ansarmughal99** [reply]
Thanks for your reply 
Just Updated App and Restore backup and got All the Saved Clipboard
